haiku-apps/melt/melt-1.3.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A CD burning application"
+DESCRIPTION="Melt, a CD burning application."
```

Comment:
Shall we change this something a bit different from what we already have for the `SUMMARY`?
```
DESCRIPTION="Melt is a frontend application which uses mkisofs and cdrecord, \
both from the cdrtools project, to create and burn Audio and Data CDs."
```

Reply:
Thanks for the much better description!